financial/simple_moving_average.py

```diff
@@ -0,0 +1,59 @@
+"""
+Calculate the Simple Moving Average (SMA) for a time series data.
+https://en.wikipedia.org/wiki/Moving_average
+"""
+
+
+def simple_moving_average(data: list[int], window_size: int) -> list[float | None]:
```

Comment:
```suggestion
def simple_moving_average(data: list[float], window_size: int) -> list[float | None]:
```
SMA also works with non-integer numerical data